nginx-autoinstall.sh

```diff
@@ -243,6 +247,12 @@ case $OPTION in
 		cd /usr/local/src/nginx/modules || exit 1
 		git clone --depth 1 https://github.com/FRiCKLE/ngx_cache_purge
 	fi
+	
+	# Nginx Substitutions Filter
+	if [[ $SUBFILTER == 'y' ]]; then
+		cd /usr/local/src/nginx/modules || exit 1
+		git clone https://github.com/yaoweibin/ngx_http_substitutions_filter_module
```

Comment:
Please use a tag 🙏 https://github.com/yaoweibin/ngx_http_substitutions_filter_module/releases

Reply:
It's been 13 commits to master since this [tag](https://github.com/yaoweibin/ngx_http_substitutions_filter_module/compare/v0.6.4...master)